plugins/Community_Basic.js

```diff
@@ -0,0 +1,86 @@
+/*:
+ * @plugindesc Basic plugin for manipulating important parameters..
+ * @author RM CoreScript team
+ *
+ * @param cacheLimit
+ * @desc The upper limit of images' cached size (MPixel)
+ * @default 20
+ *
+ * @param screenWidth
+ * @desc The resolution of screen width
+ * @default 816
+ *
+ * @param screenHeight
+ * @desc The resolution of screen height
+ * @default 624
+ *
+ * @param renderingMode
+ * @desc The rendering mode (canvas/webgl/auto)
+ * @default auto
+ *
+ * @param alwaysDash
+ * @desc The initial value whether the player always dashes (on/off)
+ * @default off
+ */
+
+/*:ja
+ * @plugindesc 基本的なパラメーターを設定するプラグインです。
+ * @author RM CoreScript team
+ *
+ * @param cacheLimit
+ * @desc 画像のメモリへのキャッシュの上限値 (MPix)
+ * @default 20
+ *
+ * @param screenWidth
+ * @desc 画面サイズの幅
+ * @default 816
+ *
+ * @param screenHeight
+ * @desc 画面サイズの高さ
+ * @default 624
+ *
+ * @param renderingMode
+ * @desc レンダリングモード (canvas/webgl/auto)
+ * @default auto
+ *
+ * @param alwaysDash
+ * @desc プレイヤーが常時ダッシュするかどうかの初期値 (on/off)
+ * @default off
+ */
+
+(function() {
+    function toNumber(str, def) {
+        return isNaN(str) ? def : +(str || def);
+    }
+
+    var parameters = PluginManager.parameters('Community_Basic');
+    var cacheLimit = toNumber(parameters['cacheLimit'], 20);
+    var screenWidth = toNumber(parameters['screenWidth'], 816);
+    var screenHeight = toNumber(parameters['screenHeight'], 624);
+    var renderingMode = parameters['renderingMode'].toLowerCase();
+    var alwaysDash = parameters['alwaysDash'].toLowerCase() === 'on';
+
+    ImageCache.limit = cacheLimit * 1000 * 1000;
+    SceneManager._screenWidth = screenWidth;
+    SceneManager._screenHeight = screenHeight;
+    SceneManager._boxWidth = screenWidth;
+    SceneManager._boxHeight = screenHeight;
+
+    SceneManager.preferableRendererType = function() {
```

Comment:
option = webgl, renderingMode = canvas => canvas
option = canvas, renderingMode = webgl => canvas
Did you intent that?